includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -1174,10 +1174,29 @@ public function add_last_activity_to_user_query( WP_User_Query $query ) {
 			WHERE {$wpdb->comments}.user_id = {$wpdb->users}.ID
 			AND {$wpdb->comments}.comment_approved IN ('complete', 'passed', 'graded')
 			AND {$wpdb->comments}.comment_type = 'sensei_lesson_status'
-			ORDER BY {$wpdb->comments}.comment_date_gmt DESC
 		) AS last_activity_date";
 	}
 
+	/**
+	 * Order query based on the custom field.
+	 *
+	 * @since  4.3.0
+	 * @access public
```

Comment:
I think this method should be marked as private to discourage dev from using it.

Reply:
Yeah, you are right, hooks don't work with private functions, so we have to make them public. But to discourage the devs from using them as a normal public function, we change the "@access public" to "@access private" in the doc comment above the actual function.

Reply:
Oh I saw you did it in one function, taught it was mistake :D